specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-12-01/bms.json

```diff
@@ -5485,6 +5500,13 @@
           "$ref": "#/definitions/RetentionPolicy",
           "description": "Retention policy with the details on backup copy retention ranges."
         },
+        "tieringPolicy": {
+          "description": "Tiering policy to automatically move RPs to another tier\r\nKey is Target Tier, defined in RecoveryPointTierType enum.\r\nTiering policy specifies the criteria to move RP to the target tier.",
```

Comment:
>Tiering policy specifies the criteria t [](http://example.com/codeflow?start=148&length=39)

Are there any combination of source and target tiers that are not allowed by the service ? If so how does the end user discover such combinations?

Reply:
There is only one source tier and one target tier as of now

Reply:
Please specify the conditions inline or add a link to the documentation. Also when you add support for more combinations, it will be considered a breaking change and you will have to release that as a part of a new API version.

Reply:
The documentation is not available yet. I'll add the conditions inline.

Yes, new combinations will be added in a new API version.